index.html

```diff
@@ -1064,18 +1064,21 @@ <h2>Thing Description Server</h2>
                             MUST serve that resource with a `GET` method.
                         </span>
                         <span class="rfc2119-assertion" id="service-http-resp">
-                            A successful response from
-                            an HTTP-based TD Server providing a <a>TD</a> 
-                            MUST have a 200 (OK) status, contain a `application/td+json`
-                            Content-Type header, and the <a>TD</a> in the body.
-                        </span>
+                            A successful response from an HTTP-based TD Server providing a <a>TD</a>
+                            MUST have 200 (OK) status and the <a>TD</a> in the body.</span>
+                        <span class="rfc2119-assertion" id="service-http-resp-content-type">
+                            A successful response with JSON serialization MUST contain `application/json`
```

Comment:
The use of "contain" is a bit strange here.  I can understand it if I interpret the content-types and defining sets, and td+json is a subset of json, but perhaps there is a better word than "contain" to make this clearer.

Reply:
Should we instead be more explicit and say "MAY" and "SHOULD" for the various options?